### PR TITLE
use chainguard set of actions to avoid duplication

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
- use chainguard set of actions to avoid duplication

#### Ticket Link
n/a

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: .github/workflows/fulcio-rekor-kind.yaml [hunk 1]
```diff
@@ -94,13 +86,20 @@ jobs:
...
+      id: kind
+      with:
+        k8s-version: ${{ matrix.k8s-version }}
+        registry-authority: registry.local:5000
+        cluster-suffix: cluster.local
+
+    - name: Setup Knative
+      uses: chainguard-dev/actions/setup-knative@main
+      with:
+        version: ${KNATIVE_VERSION}
```
Comment (mattmoor) [inline .github/workflows/fulcio-rekor-kind.yaml:99]: Does this not need `${{ env.KNATIVE_VERSION}}`? 🤔

**Review by mattmoor — COMMENTED** [review]
cc @vaikas

File: .github/workflows/fulcio-rekor-kind.yaml [hunk 2]
```diff
@@ -94,13 +86,20 @@ jobs:
...
+        serving-features: '{"kubernetes.podspec-fieldref":"enabled", "kubernetes.podspec-volumes-emptydir":"enabled", "kubernetes.podspec-init-containers": "enabled", "kubernetes.podspec-securitycontext":"enabled"}'
+        serving-defaults: '{"revision-timeout-seconds":"5","container-concurrency":"100"}'
```
[the remark this answers is not in the document]
Comment (cpanato) [reply .github/workflows/fulcio-rekor-kind.yaml:101]: ok

File: .github/workflows/test-release.yaml [hunk 3]
```diff
@@ -34,22 +34,23 @@ jobs:
...
-      run: |
-        curl -Lo ./setup-kind.sh https://github.com/sigstore/scaffolding/releases/download/${{ env.RELEASE_VERSION }}/setup-kind.sh
-        chmod u+x ./setup-kind.sh
-        ./setup-kind.sh \
-          --registry-url $(echo ${KO_DOCKER_REPO} | cut -d'/' -f 1) \
-          --cluster-suffix cluster.local \
-          --k8s-version ${{ matrix.k8s-version }} \
-          --knative-version ${KNATIVE_VERSION}
+      uses: chainguard-dev/actions/setup-kind@main
+      id: kind
```
[the remark this answers is not in the document]
Comment (cpanato) [reply .github/workflows/test-release.yaml:41]: yes, but also that does not hurt :)

File: .github/workflows/fulcio-rekor-kind.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (cpanato) [reply .github/workflows/fulcio-rekor-kind.yaml]: done

File: .github/workflows/test-release.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (mattmoor) [reply .github/workflows/test-release.yaml:40]: Ack, just making sure I'm not missing anything!

**Comment by cpanato** [comment]
> LGTM, but I'll defer to @vaikas for scaffolding!
> 
> Maybe we can also clean up the `setup-kind` script here now? Is it used elsewhere?

was thinking the same, but was not sure if that is used outside the actions, maybe @vaikas can comment on that

**Review by vaikas — APPROVED** [review]
Thank you so much!!!

**Pull request merged**